src/modules/esl-trigger/core/esl-trigger.ts

```diff
@@ -44,6 +44,11 @@ export class ESLTrigger extends ESLBaseElement {
   /** Selector of inner target element to place aria attributes. Uses trigger itself if blank */
   @attr({defaultValue: ''}) public a11yTarget: string;
 
+  /** Value of aria-label for active state */
+  @attr({defaultValue: ''}) public a11yLabelActive: string;
```

Comment:
```suggestion
  @attr() public a11yLabelActive: string;
```
Suggested example is tottaly equal even in terms of upcoming `@attr` update